content/en/internal/configuration-database/access/_index.md

```diff
@@ -1,64 +1,65 @@
 ---
 title: "Access"
-description: "Accessing GOCDB"
-weight: 20
+description: "Accessing the configuration database"
+weight: 25
 type: "docs"
 ---
 
-The GOCDB can be accessed at [https://goc.egi.eu](https://goc.egi.eu).
+To access [web interface](https://goc.egi.eu) of the EGI configuration database
+(GOCDB), users can either:
 
-To access the web interface, users can:
+- [Use EGI Check-in](#using-institutional-account-via-egi-check-in) to access
+  the GOCDB with an institutional account, or
```

Comment:
```suggestion
  the Configuration Database with an institutional account, or
```

We should favour the branded service name (Configuration Database) over the legacy/tool name (GOCDB).

Reply:
Fixed